src/Microsoft.AspNet.Identity/IdentityLogger.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.CompilerServices;
+using Microsoft.AspNet.Identity;
+
+namespace Microsoft.Framework.Logging
```

Comment:
My pushback is not against the pattern in general with Infrastructure/Internal, my pushback is that this isn't internal code we happen to make public.  Its a valid extensibility scenario for custom managers to plug in their own logging and use these extensions, so from that perspective it deserves a proper real namespace like Microsoft.AspNet.Identity.Logging, not a funky (don't use this since it has infrastructure/internal) namespace...